### PR TITLE
Securingdev readme update

### PR DESCRIPTION
**Pull request by securingdev**
Since I've been contributing lately, thought I'd add myself to the README.

### CODE REVIEW

**Review by felickz — APPROVED** [review]
Welcome aboard 😅

File: codeql
Comment (felickz) [inline codeql:1]: meaning to pull in the submodule  ⬆️  ?  (unclear if it should be accompanied by additional runs or if automation handles all the checks)

**Review by GeekMasher — APPROVED** [review]
Welcome!

File: codeql
Comment (GeekMasher) [reply codeql:1]: Agree, be careful updating the submodule as this is done automatically and you can be out of sync

**Pull request merged**